transcribe.py

```diff
@@ -250,6 +262,7 @@ def main(args):
 	parser.add_argument('--fp16', choices = ['O0', 'O1', 'O2', 'O3'], default = None)
 	parser.add_argument('--num-workers', type = int, default = 0)
 	parser.add_argument('--mono', action = 'store_true')
+	parser.add_argument('--audio-backend', default = 'ffmpeg', choices = ['sox', 'ffmpeg'])
```

Comment:
please remove default setting, because it would use ffmpeg external process even for wav files. for now we rely on None default backend for some default choice